commands/guessnationality.js

```diff
@@ -0,0 +1,58 @@
+const { SlashCommandBuilder } = require("discord.js")
+const axios = require("axios").default
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("guessnationality")
+    .setDescription("Guess the nationality of a random name.")
+    .addStringOption((option) =>
+      option
+        .setName("name")
+        .setDescription("Guess the nationality of a specific name.")
+        .setRequired(false)
```

Comment:
Please change the name option to required true

Reply:
Hey @thieleju, I can totally do this, however, the reason I set it to optional is that if the user doesnt provide a name, a random name is provided by the random-data-api.

Reply:
Oh right, I missed that. I thing we could change the description to "Guess the nationality of specific name or fetch a random name and its nationality".

What do you think?

Reply:
That sounds good to me! I will make that change today and push it to my branch. Thank you @thieleju :)